### DISCUSSION TITLE
Keep in touch with WHAM developments

### DISCUSSION THREAD

**Discussion by ChristineStawitz-NOAA** [NOAA-FIMS/collaborative_workflow#126] [General]

**Comment by timjmiller** [comment]
new wham release 1.0.6 a few weeks ago includes random effects and covariate effects on recruitment and one-step-ahead residuals for age composition observations.

ongoing work to extend to multi-area/stock models on wham "lab" branch. Needed for black sea bass and cod research track assessments in the NEUS. Very early stages.
